### PR TITLE
UI fixes

### LINKED ISSUE #473: [BUG]: Some UI and Code fixes required on the table.html and info.html pages
**Describe the bug**
Some refactoring and fixes required in table.html and info.html page routes.

**To Reproduce**
1. Go to [landing page](https://wolf.cci.emory.edu//camic_org/apps/landing/landing.html)
2. Click on MORE and Sign Up via Google 
3. You will be redirected to [table.html](https://wolf.cci.emory.edu//camic_org/apps/table.html) page.
4. See errors.

What are major errors?
* When Page Loads the data(if there) isn't fetched. And so, the footer due to its CSS styling isn't at its correct position. This makes UI look bad and can be fixed.
* Click on info button to go to [info.html](https://wolf.cci.emory.edu//camic_org/apps/Info.html) and it can be observed that Info and Slide both are active links.
* There are some redundancies in the code and can be refactored as an easy non-breaking fix.

**Expected behavior**
* The Footer must be at its place, by setting CSS styles.
* Navbar Active link must be consistent.

**Screenshots**
active_link
footer

**Desktop (please complete the following information):**
 - OS: Windows 10, Ubuntu Linux
 - Browser: Chrome

**Smartphone (please complete the following information):**
 - Device: Moto Z Play
 - OS: Android 7.1.1 
 - Browser Chrome

### PR DESCRIPTION
**Pull request by sohamsshah**
## Description
UI fixes as mentioned in #473 are as follows:

UI Fixes:
* Footer in both pages viz. table.html and info.html are properly placed at bottom
* The 'Slide' button used to be active in Navbar of info.html page. This has been fixed in this PR.

---

Code Changes:
* All the html files in the codebase has been named in lowercase. To keep the naming convention consistent, the name of 'Info.html' has been changed to 'info.html'
* In table.html, the CSS was written in <style> tag, while there exists a CSS file for it. So, made changes accordingly. 
* Also used 'Semantic HTML' `footer` tag instead of `div` and made corresponding changes in the html and css files.

## Motivation and Context
**Why is this change required? What problem does it solve?
- It fixes #473 
- It helps to solve minor UI bugs in the codebase.

## How Has This Been Tested?
**Please describe in detail how you tested your changes.
- Configured Dev Environment as suggested in the Developer Guide.
- Tested on Localhost with responsiveness check too 

## Types of changes
**What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
**Go over all the following points, and put an `x` in all the boxes that apply.
**(If you're unsure about any of these, don't hesitate to ask. We're here to help!)
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Let me know how this sounds. Will make changes if needed ASAP. Thank you for reviewing!

### CODE REVIEW

File: apps/Info.html [hunk 1]
```diff
@@ -164,9 +164,9 @@ <h3 class="text-center h3 mb-2" style="margin-top:8px;margin-bottom:10px;">Infor
 	</div>
 </div>
 
-<div class="text-center text-white bg-dark p-3 mt-2 footer" style="position: static;bottom: 0;width: 100%;">
+<footer class="text-center text-white bg-dark p-3 mt-2">
 	<p class="p">Copyright © 2020 caMicroscope</p>
```
Comment (birm) [inline apps/Info.html:168]: Looks good to me!
Just, while you're at it, welcome to 2021!
```suggestion
	<p class="p">Copyright © 2021 caMicroscope</p>
```

**Review by nanli-emory — APPROVED** [review]

File: apps/Info.html (hunk 1 above)
Comment (sohamsshah) [reply apps/Info.html:168]: Haha yes, good suggestion. Adding it😊
Thanks for the review!

**Comment by birm** [comment]
I've fixed the conflicts, just going to test to make sure I didn't mess anything up.

File: apps/Info.html [hunk 2]
```diff
@@ -60,9 +60,12 @@
 				<li class="nav-item link">
 					<a class="nav-link" href="landing/landing.html"> <i class="fas fa-home"></i> Home</a>
 				</li>
-				<li class="nav-item link active">
+				<li class="nav-item link">
 					<a class="nav-link" href="table.html"> <i class="fas fa-list-ul"></i> Slides</a>
 				</li>		
+				<li class="nav-item active link">
+				<a class="nav-link" href="info.html"> <i class="fas fa-info-circle"></i> Info</a>
```
Comment (birm) [inline apps/Info.html:67]: I'm getting a 404 on all links to info.html; it appears to be case sensitive. we can rename this file to info.html or change the links back to Info.html

**Review by birm — CHANGES_REQUESTED** [review]

File: apps/Info.html (hunk 2 above)
Comment (birm) [reply apps/Info.html:67]:
```suggestion
				<a class="nav-link" href="Info.html"> <i class="fas fa-info-circle"></i> Info</a>
```

File: apps/landing/landing.html [hunk 3]
```diff
@@ -50,6 +50,9 @@
             <li class="nav-item  link" style="font-family: sans-serif;">
                 <a class="nav-link" href="../table.html"> <i class="fas fa-list-ul"></i>  Slides</a>
             </li>
+            <li class="nav-item link">
+                <a class="nav-link" href="../info.html"> <i class="fas fa-info-circle"></i> Info</a>
```
Comment (birm) [inline apps/landing/landing.html:54]:
```suggestion
                <a class="nav-link" href="../Info.html"> <i class="fas fa-info-circle"></i> Info</a>
```

File: apps/table.html [hunk 4]
```diff
@@ -70,6 +59,9 @@
 				<li class="nav-item active link">
 					<a class="nav-link" href="table.html"> <i class="fas fa-list-ul"></i> Slides</a>
 				</li>
+				<li class="nav-item link">
+					<a class="nav-link" href="info.html"> <i class="fas fa-info-circle"></i> Info</a>
```
Comment (birm) [inline apps/table.html:63]:
```suggestion
					<a class="nav-link" href="Info.html"> <i class="fas fa-info-circle"></i> Info</a>
```

File: apps/table.html [hunk 5]
```diff
@@ -142,13 +134,25 @@ <h1 class="h1">caMicroscope</h1>
 			<div style="align-content: center;">
 				<div class="btn-group" role="group">
 				<button type="button" style="border-color: white;" class="btn btn-secondary bg-white text-dark" title="Slides Table"><i class="fas fa-list-alt"></i>  Slides</button>
-				<a href="./Info.html">	<button style="border-color: white; border-radius: 0 5px 5px 0;" type="button" class="btn btn-secondary bg-info text-light" title="Information Dashboard"><i class="fas fa-info-circle"></i>  Info</button> </a></div>
+				<a href="./info.html">	<button style="border-color: white; border-radius: 0 5px 5px 0;" type="button" class="btn btn-secondary bg-info text-light" title="Information Dashboard"><i class="fas fa-info-circle"></i>  Info</button> </a></div>
```
Comment (birm) [inline apps/table.html:137]:
```suggestion
				<a href="./Info.html">	<button style="border-color: white; border-radius: 0 5px 5px 0;" type="button" class="btn btn-secondary bg-info text-light" title="Information Dashboard"><i class="fas fa-info-circle"></i>  Info</button> </a></div>
```

Comment (sohamsshah) [reply apps/table.html]: Really good point. I actually changed all the links to info.html after changing the name to 'info.html' in order to maintain the naming convention. But due to some reason (maybe I forgot) it didn't reflect in the PR. Shall I rename it and commit again, or change all the links to Info.html back? What do you prefer?

File: apps/info.css [hunk 6]
```diff
@@ -1,10 +1,16 @@
...
-			flex-direction: column;
+	width: 100%;
+	height: 100%;
+	min-height: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+footer{
+  position: absolute;
```
Comment (birm) [inline apps/info.css:11]: Having an issue with this; it does not stay at the bottom of the screen like I presume you intend, but when scrolling down, it stays fixed to the document. That is:
image
 to 
image

Comment (birm) [reply apps/info.css:11]: (worth noting, that to me at least, the copyright notice isn't at all important, certainly not enough for it to stick eternally to the user's window)

Comment (sohamsshah) [reply apps/info.css:11]: Thank you for the review, looking into this issue.

Comment (sohamsshah) [reply apps/info.css]: Hey, I have solved the problem by using flexbox css. I wrapped the content with a flex column container and justified content as 'space-between'. I hope this works fine now :)

**Comment by birm** [comment]
Thanks for being so responsive to the review!

**Pull request merged**